matRad_calcParticleDose.m

```diff
@@ -169,24 +175,14 @@
             % reasons
             energyIx = max(round2(stf(i).ray(j).energy,4)) == round2([machine.data.energy],4);
             
+            maxLateralCutoffDoseCalc = max(machine.data(energyIx).LatCutOff.CutOff);
             
-            maxLateralCutoff = max(machine.data(energyIx).LatCutOff.CutOff);
-            
-            
-            % Ray tracing for beam i and ray j
-            [ix,radDepths,~,latDistsX,latDistsZ] = matRad_calcRadGeoDists(ct.cube, ...
-                                                        V,...
-                                                        pln.isoCenter, ...
-                                                        rot_coordsV, ...
-                                                        ct.resolution, ...
-                                                        stf(i).sourcePoint, ...
-                                                        stf(i).ray(j).targetPoint, ...
-                                                        stf(i).sourcePoint_bev,...
-                                                        stf(i).ray(j).targetPoint_bev, ...
-                                                        coordsV, ...
-                                                        maxLateralCutoff, ...
-                                                        visBool);
-            
+            % Ray tracing for beam i and ray j                          
+            [ix,latDistsX,latDistsZ] = matRad_calcGeoDists(rot_coordsV, ...
+                                                       stf(i).sourcePoint_bev, ...
+                                                       stf(i).ray(j).targetPoint_bev, ...
+                                                       maxLateralCutoffDoseCalc);
+            radDepths = radDepthCube(V(ix));
             radialDist_sq = latDistsX.^2 + latDistsZ.^2;    
```

Comment:
We could also compute this quantity within matRad_calcGeoDists if we ask for four outputs! They are computed anyways!